src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline

```diff
@@ -4319,6 +4319,7 @@ import six
 def sample_discuss_book(image_file_name):
   """Testing calling forms"""
```

Comment:
This comes from `SampleValueSet.Description`. It seems the field was used as the description of the sample function for Python, but ignored by other languages. I added it to other languages as well. I _think_ this is the intended behavior?

Reply:
Not exactly, but this is fine. (See comment in my review.)